tests/test_grouping.py

```diff
@@ -19,9 +21,11 @@ def test_grouping_parenthesis():
     assert len(parsed.tokens[2].tokens[3].tokens) == 3
 
 
-def test_grouping_comments():
+@pytest.mark.parametrize('options', [{'sql_dialect': 'Default'},
```

Comment:
We should also add tests where options is an empty dictionary (the current behavior). OTOH 'Default' shouldn't be tested as it's just used internally when sql_dialect isn't given.

I'm not really happy how parametrize is used here. When we add other options than sql_dialect it gets really tricky to test all combination of options this way. But I'll have to peak into the pytest docs, there was a way to build such parameter matrices in pytest.